deploy/grafana-dashboard/utils/color_mapping.py

```diff
@@ -19,3 +21,9 @@ def color_mapping(name, color):
 
 
 grey_all_mapping = color_mapping("all", "#4c4b5a")
+
+color_mapping_environments = {}
```

Comment:
Should we include color mappings for all environments? Depending on the graph, the others might be green or just have another color

<img width="753" alt="Bildschirm­foto 2022-11-10 um 00 13 57" src="https://user-images.githubusercontent.com/7300329/200962209-d2d23ce8-8a6a-4e2d-96f4-7f7152eacdf1.png">